src/components/Common/SearchBox.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, forwardRef } from 'react';
+import remcalc from 'remcalc';
+import styled, { keyframes } from 'styled-components';
+import is from 'styled-is';
+
+import { Input } from './Forms';
+import Anchor from './Anchor';
+
+const fadeIn = keyframes`
+  from{
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const SearchField = styled.section`
+  position: relative;
+  margin-bottom: ${remcalc(36)};
+`;
+
+export const Results = styled.ul`
+  width: 100%;
+  max-height: ${remcalc(200)};
+  list-style-type: none;
+  background: ${props => props.theme.colors.greyBg};
+  position: absolute;
+  left: 0;
+  top: ${remcalc(56)};
+  z-index: 10;
+  overflow-x: auto;
+  overflow-y: scroll;
+  will-change: opacity;
+  animation: ${fadeIn} ${props => props.theme.animations.long} linear;
+`;
+
+export const Result = styled.li`
+  padding: ${remcalc(8)} ${remcalc(24)};
+  font-size: ${remcalc(18)};
+  line-height: ${remcalc(24)};
```

Comment:
use theme values

Reply:
will do it but will leave the text properties like that, because we don't have theme values for that